Gemfile

```diff
@@ -1,10 +1,10 @@
 source 'https://rubygems.org'
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.6.6'
 
 # Service/framework dependencies
-gem 'rails', github: 'rails/rails', branch: 'master' # , '~> 6.0.2', '>= 6.0.2.2'
+gem 'rails', git: 'http://github.com/rails/rails.git', ref: "f4bf62ee8e0f8762a82f85a789119b2211fdecb4"
```

Comment:
I've locked us to a commit for now. I'll probably run against master when I'm working, but I think it's better for not needing to constantly update the gems to fix a commit.

Reply:
Makes sense